github_rules/github_repo_initial_access.py

```diff
@@ -3,7 +3,13 @@
 
 def rule(event):
     if event.get("action").startswith("git.") and not event.get("repository_public"):
-        # trigger on any of the git actions, http or ssh
+        
+        # if the actor field is empty, short circuit the rule
+        actor = event.get("actor")
+        if actor == '' or actor == None:
+            return True
```

Comment:
We use the data model/`.udm("actor_user")` in the title, I wonder if we could use it here also? I think this means to return False/not trigger an alert? Also, another option that would shrink this down a bit: 

```suggestion
        if not event.udm("actor_user"):
            return False
```

Reply:
Good point - adopted yours